org.planqk.nisq.analyzer.core/src/main/java/org/planqk/nisq/analyzer/core/web/dtos/requests/CompilerSelectionDto.java

```diff
@@ -37,4 +37,6 @@
     URL circuitUrl;
 
     String token;
+
+    String bearerToken;
```

Comment:
the bearer/refreshToken must also be considered in the CompilerSelectionController